index.html

```diff
@@ -38,8 +41,190 @@ <h1>I'm Mucha <br> Glad to see <br> you!</h1>
                 <li><a href="#"><img src="images/medium.svg" alt="medium"> </a></li>
             </ul>
         </section>
+
+
+    </div>
+
+    <!-- card 1 -->
+    <div class="work-container">
+        <div class="portfolio-img">
+            <img src="images/Snapshoot-Portfolio-1.png" alt="">
+        </div>
```

Comment:
- [ ] Again, remember we have to use [Semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp) elements to ensure that our code is professional 👏 . Therefore, I suggest that you  ensure that the container of the work container section is a `section` tag 👍 .